spec/app/validators/password_is_strong_validator_spec.rb

```diff
@@ -231,7 +216,7 @@
 
             context 'without repetition' do
               let(:value) do
-                'A$uperg00dp@ssw0rd'
+                'A$uperg0dp@sw0rd'
```

Comment:
Does this need to change?  Testing shows this original value should still be valid.

Reply:
Based on the way contains_repetition? is written it can not have two characters next to each other. I was assuming that was correct, but am guessing that the contains_repetition? is broken.

Reply:
Contains repetition says you cannot repeat two character sequences not that a password cannot repeat any character.

>    `# Password repetition (quite basic) -- no "aaaaaa" or "ababab" or "abcabc" or`
>    `# "abcdabcd" (but note that the user can use "aaaaaab" or something).`

Reply:
To be clear `contains_repetition?`requires the whole password to be repetitions of 1 to 4 characters to fail.  A more descriptive name for the method would be `is_only_repetition?`

Failing examples would be
```
aaaaaaaa
asasasas
asdasdasd
asdfasdfasdfasdf
```
Passing is a simple as adding 1 different character anywhere in the string:
```
aaaaaaaa1
asasasas1
asdasdasd1
asdfasdfasdfasdf1
```